src/docs/markdown/download.md

```diff
@@ -58,13 +58,42 @@ RHEL/CentOS 7:
 [**View the Caddy COPR**](https://copr.fedorainfracloud.org/coprs/g/caddy/caddy/)
 
 
+## Linux & Raspberry Pi
```

Comment:
Why do we need to distinguish Linux from Raspberry Pi?

Reply:
I often do in my own documentation for a couple of reasons. It's kind of a "Kleenex" vs "Tissue Paper" king of thing.

Generally speaking "Linux" means not "kernel" but rather "something Debian-ish on AMD64 hardware for web services or general desktop nerdery" whereas "Raspberry Pi" means "Internet of Things, but easier than arduino, on ARM".

Someone searching for "Linux" is a different user than someone searching for "Raspberry Pi".

Also, many modern tutorials for Linux don't readily work on Raspberry Pi because they're based on different sets of tools due to the difficulty of getting everything on an RPi.

I can drop it if you like, but including both terms is a signal to the reader that yes, in fact, Caddy is a product that works on Raspberry Pi (ARM Linux), and Webi in an installer that works there too.